packages/core/lib/segments/segment_utils.js

```diff
@@ -45,6 +46,27 @@ var utils = {
 
   getStreamingThreshold: function getStreamingThreshold() {
     return utils.streamingThreshold;
+  },
+
+  /**
+   * Parses an HTTP response object to return an X-Ray compliant HTTP response object.
+   * @param {http.ServerResponse} res
+   * @returns {Object} - X-Ray response object to be added to (sub)segment
+   */
+  getHttpResponseData: (res) => {
+    const ret = {};
+    if (!res) {
+      return ret;
+    }
+    
+    const status = safeParseInt(res.statusCode);
+    if (status !== 0) {
+      ret.status = status;
+    }
+    if (res.headers && res.headers['content-length']) {
+      ret.content_length = safeParseInt(res.headers['content-length']);
```

Comment:
Should we keep recording the `content-length` even if it's `0`?

Reply:
Yes we should IMO, `content-length` can be 0 if there is a response with an empty body, which could still be valuable for a customer to know.